ci/playbooks/collect-logs.yml

```diff
@@ -69,12 +69,19 @@
               pip3 freeze >> ./python.log;
               dmesg -T > ./dmesg.log;
 
+        - name: Generate list of logs to collect in home directory
+          ansible.builtin.find:
+            paths: "{{ ansible_user_dir }}"
+            patterns: "*.log"
```

Comment:
are you expecting other files here? if not why not explicitly dlrn.log  ... or is the name not predictable somehow? 
fine for collecting other files just that the commit message explicitly mentions dlrn log

Reply:
Thanks Marios, first I thought we just weren't collecting the dlrn.log file... Then I found a bigger issue while I was just testing it today, I'll update the commit message now :)

Reply:
To answer your question if you compare two runs of the job you can see the logs that weren't being collected: 
https://logserver.rdoproject.org/28/128/1b4f01dfb0c152eb12924bbff765495c63262126/github-check/tcib-build-containers/a6b3df0/

https://logserver.rdoproject.org/28/128/1b4f01dfb0c152eb12924bbff765495c63262126/github-check/tcib-build-containers/9c737d6/